action: implemented and added README (#1)
ariga/setup-atlas: README.md

# setup-action

> A GitHub Action for installing up [Atlas](https://github.com/ariga/atlas).

Reviewer: installing or setting up but not installing up :-)

> This action is used for installing Atlas on the Github runner.

Reviewer: Duplicate, choose either this or the first one

> > Please note that for most use cases, you should first consider using [Atlas Action](https://github.com/ariga/atlas-action) which can save you a lot of work.

Reviewer: `atlas-action` is used solely for CI so it is not necessarily "most cases". Instead I would say that we have other actions link [here](https://atlasgo.io/integrations/github-actions) and maybe list them.

> > Please note that for most use cases, you should first consider using [Atlas Action](https://github.com/ariga/atlas-action/tree/master) which can save you a lot of work.

Reviewer 2: ```suggestion
> Please note that for most use cases, you should first consider using [Atlas Action](https://github.com/ariga/atlas-action) which can save you a lot of work.
```

### Usage

> Add `.github/workflows/setup-atlas.yaml` to your repo with the following contents:

Reviewer: Since state isn't shared between workflows, this action is only useful as a first step in a workflow that uses Atlas, so `setup-atlas.yaml` probably will not be its name. You can show something like `deploy-schema.yaml` that installs atlas and then runs `atlas schema apply --env prod` or similar

```yaml
name: My Action Name
run-name: I'm using Atlas 🚀
on: [push]
jobs:
  use-atlas:
    runs-on: ubuntu-latest
    steps:
      - uses: ariga/setup-atlas@master
      - run: atlas version
```

> #### version

Reviewer: ```suggestion
### Inputs

#### `version`
```

# setup-action

A GitHub Action for installing [Atlas](https://github.com/ariga/atlas).

> > Please note that other [atlas-actions](https://atlasgo.io/integrations/github-actions) might suit your needs better as they can do more heavy lifting operations

Reviewer: ```suggestion
> See the docs for other [Atlas GitHub Actions](https://atlasgo.io/integrations/github-actions) for operations such as linting, syncing, or deploying your migration directory. 
```
